interface.html

```diff
@@ -5,30 +5,7 @@
     </div>
     <template v-else>
       <div class="form-group">
-        <div class="col-sm-4 control-label">
-          <label>{{settings.dataSourceLabel}}</label>
-        </div>
-        <div class="col-sm-8">
-          <label class="select-proxy-display">
-            <select data-label="select" class="hidden-select form-control" v-model="selectedDataSource"
-                    :disabled="!dataSources">
-              <option value="null">-- \{{ dataSources ? 'Select a data source' : 'Please wait' }}</option>
-              <option disabled>------</option>
-              <option value="new">Create a new data source</option>
-              <option disabled>------</option>
-              <option :value="dataSource" v-for="dataSource in dataSources">\{{dataSource.name}}</option>
-            </select>
-            <span class="icon fa fa-chevron-down"></span>
-            <span class="select-value-proxy">
-              <template v-if="dataSources">
-                <template v-if="selectedDataSource">\{{selectedDataSource.name}}</template>
-                <template v-else>-- Select a data source</template>
-              </template>
-              <template v-else>-- Please wait</template>
-            </span>
-          </label>
-          <small v-if="manageDataBtn"><a href="#" v-on:click.prevent="manageDataSource(selectedDataSource.id)">Edit data source</a></small>
-        </div>
+        <div id="dataSourceProvider" class="col-sm-8 ui-sortable"></div>
```

Comment:
1. It should use `col-sm-12` because the original `col-sm-4` and `col-sm-8` together occupy 12 columns. We should add `col-sm-4` for the label and `col-sm-8` for the rest in the data source provider interface. [Ref](https://github.com/Fliplet/fliplet-widget-data-source-provider/blob/a2305bb5c5ee95358321b7c11badb7aa867f7957/src/DataSourceProvider.vue#L3)
1. What's `ui-sortable` for? It wasn't there in the original implementation and doesn't seem like a class that's used.

```suggestion
        <div id="dataSourceProvider" class="col-sm-12"></div>
```